gprofiler/gprofiler_types.py

```diff
@@ -32,6 +33,31 @@ class ProfileData:
 ProcessToProfileData = MutableMapping[int, ProfileData]
 
 
+class ProfilingErrorStack(StackToSampleCount):
+
+    PROFILING_ERROR_STACK_PTRN = re.compile(r".*;\[Profiling .+: .+\]")
+
+    def __init__(self, what: str, reason: str, comm: str):
+        super().__init__()
+        self.update({f"{comm};[Profiling {what}: {reason}]": 1})
+
+    @staticmethod
+    def is_error_stack(stack: StackToSampleCount) -> bool:
+        return len(stack) == 1 and ProfilingErrorStack.PROFILING_ERROR_STACK_PTRN.match(next(iter(stack))) is not None
+
+    @staticmethod
+    def attach_error_to_stacks(
+        source_stacks: StackToSampleCount, error_stack: StackToSampleCount
+    ) -> StackToSampleCount:
+        _, error_frame = next(iter(error_stack)).split(";", maxsplit=1)
+        dest_stack: StackToSampleCount = StackToSampleCount()
+        for (frame, count) in source_stacks.items():
+            l, r = frame.split(";", maxsplit=1)
```

Comment:
```suggestion
            comm, stack = frame.split(";", maxsplit=1)
```
give real names when possible :sweat_smile:

Reply:
Sure.